curriculum/challenges/english/08-coding-interview-prep/rosetta-code/knight's-tour.md

```diff
@@ -0,0 +1,142 @@
+---
+id: 5a23c84252665b21eecc7ed5
+title: Knight's tour
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+<a href="https://en.wikipedia.org/wiki/Knight%27s_tour">Problem</a>: you have a w x h chessboard, empty but for a single knight on some square. Your task is to emit a series of legal knight moves that result in the knight visiting every square on the chessboard exactly once. Note that it is <i>not</i> a requirement that the tour be "closed"; that is, the knight need not end within a single move of its start position.
```

Comment:
I'm a little confused by this description. Seems like the original challenge to to emit a series of legal moves the knight can make, but here users should return a tally of the number of initial positions from where it's possible to complete the knight's tour. We'll have to change the description to reflect that. The second sentence is what's throwing me off.

Maybe the second sentence can be changed to something like the following:

"The knight must perform a sequence of legal moves that result in the knight visiting every square on the chessboard exactly once."

What do you think?